### DISCUSSION TITLE
How to generate a new Deep Potential from two existing Deep Potentials

### DISCUSSION THREAD

**Discussion by Taixi-Feng** [deepmodeling/deepmd-kit#2780] [Q&A] [answered]
I have a problem: I generated two different graph_1.pb and graph_2.pb files using the DeepMD-kit package, and I would like to build a new graph.pb file based on the formula E = λE1 + (1 - λ)E2 to be used to run the DPMD process in the LAMMPS software. May I know how I can do this? Thanks a lot for the help!

**Comment by njzjz** [comment, marked as answer]
I added such a model in #2781.

**Reply by Taixi-Feng** [reply]
Thank you for your help.
